docs/en/04_Changelogs/4.7.0.md

```diff
@@ -76,3 +76,35 @@ An edgecase exists where a project can update to `silverstripe/framework 4.7.0`
 `silverstripe/assets 1.6.x`, and lose the Flysystem dependency entirely. The best way to avoid this
 is by ensuring you update all core modules to the new minor release at once, ideally through a core
 recipe like `silverstripe/recipe-core`.
+
+### Improved Toast notifications
+
+Toast notifications have been completely revamped
+- Toast notifications can now be triggered from React components using Redux actions
+- Actions can now be attached to the toast notifications
+- Accessibility has been improved (see the component documentation for caveats when using actions)
+
+Asset-admin has been updated to use toast notifications instead of alerts. More modules will start using them in future releases.
+
+[Review the Toast story in the pattern library](https://silverstripe.github.io/silverstripe-admin/?selectedKind=Admin%2FToasts&selectedStory=Single) to learn how and when to use the toast notifications.
+
+### DataObject constructor support for hydration
+
+Many developers wrongly assumed that the DataObject constructor could be used to "hydrate" a record by passing an array of value to it. While this worked in some cases, values would be run through setters and validation occasionally leading to inconsistencies.
+
+Silverstripe CMS Recipe 4.7 introduces improvements to the DataObject constructor to provide proper support for hydration.
+
+```php
+// Default behavior. Setters and validation will be applied to the provided data.
+// Equivalent to `Foo::create(['Title' => 'Bar'])` or `new Foo(['Title' => 'Bar'])` 
```

Comment:
I'd probably remove the `new Foo` example and only say it's equivalent to the `::create` example, or mention possible Injector related differences. My understanding was that using Injector was the preferred way anyway, so the least examples of the `new Foo` approach the better.

Reply:
Fixed